activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -120,7 +120,7 @@ def create(*arguments)
         $stderr.puts "Database '#{configuration['database']}' already exists"
       rescue Exception => error
         $stderr.puts error
-        $stderr.puts "Couldn't create database for #{configuration.inspect}"
+        $stderr.puts "Couldn't create database for #{configuration.except("username","password").inspect}"
```

Comment:
I'm torn about this. I think username and password can be useful in development mode. The issue is we don't want to leak it into the production logs. What do you think about changing this so it's only production?